package.json

```diff
@@ -19,11 +19,13 @@
     "express": "~4.12.2",
     "express-ftwebservice": "^2.0.2",
     "express-handlebars": "^2.0.1",
+    "fetch-er": "0.0.10",
     "gulp": "latest",
     "htmlparser": "^1.7.7",
     "image-type": "^2.1.0",
     "is-url-superb": "^2.0.0",
     "lodash": "^3.10.1",
+    "lodash.curry": "^3.0.2",
```

Comment:
We have the whole of lodash being pulled in as a dependency on the line above, no need for this extra dependency :-)